starter-code/css/main.css

```diff
@@ -6,3 +6,122 @@ Green: #00B172
 White: #FFF
 
 */
+
+/* --------- reset ---------- */
+
+body, p, ul, li {
+	margin: 0;
+	padding: 0;
+}
+
+
+/* --------- typography ---------- */
+
+.background-picture-container h1 {
+    color: white;
+    text-align: center;
+    padding-top: 230px;
+    font-size: 80px;
+} 
+
+
+.background-picture-container h3 {
+    color: white;
+    text-align: center;
+    font-size: 25px;
+}
+
+body {
+    font-family: Montserrat, Open+Sans;
+}
+
+/* --------- layout -------------- */
+
+.page-container {
+    padding: 0 20px 20px 20px;
+}
+
+.title-container {
+    text-align: center;
+    padding: 50px;
+    width: 250px;
+    margin: 0 auto;
+}
+
+/* --------- components ---------- */
+
+.spotify-logo {
+    float: left;
+    height: 20%;
+    width: 20%;
+}
+
+ul {
+    list-style: none;
+    float: right;
+    margin-left: 20px;
+}
+
+li {
+    margin: 30px;
+    font-size: 25px;
+}
+
+.nav-options {
+    overflow: hidden;
+    background-color: white;
+}
+
+.background-picture {
+    background-image: url(/Users/pedroneto/Desktop/ironhack/week1/lab-css-spotify-clone/starter-code/images/landing.jpg);
+    background-size: cover;
+}
+
+.background-picture-container {
+    margin: 0 -50px 0 -50px;
+    height: 900px;
+}
+
+.color-green {
+    color: green;
+}
+
+.images-advertising {
+    height: 50%;
+    width: 50%;
+}
+
+.advertising-container {
+    width: 400px;
```

Comment:
Try to use more relative units in sizing. Instead of using pixels, try to use %, vh, vw, em...